src/plots/plot.ts

```diff
@@ -80,6 +95,7 @@ export class Plot extends Component {
     super._setup();
     this._renderArea = this.content().append("g").classed("render-area", true);
     this.datasets().forEach((dataset) => this._createNodesForDataset(dataset));
+    this._createCanvas();
```

Comment:
less code, less complexity in the class hierarchy, we conceptually only need to start thinking about "svg vs canvas" down at the drawer level and the rest of the interfaces aren't affected